tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -222,10 +246,29 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 			err = CheckClusterDomainEffect(cli, []*v1alpha1.TidbCluster{tc1, tc2, tc3})
 			framework.ExpectNoError(err, "failed to check status")
 		})
-
 	})
 })
 
+func CheckPeerMembersAndClusterStatus(clusterCli ctrlCli.Client, tc *v1alpha1.TidbCluster, expectNonExistedTc *v1alpha1.TidbCluster) error {
```

Comment:
a little hard to understand `expectNonExistedTc` arg 😂

Reply:
maybe we can modify the function `CheckClusterDomainEffect` to have a detailed check. 🤔 
What's about ensuring that every member is defined in TC?

Reply:
This function is trying to ensure that tc3 doesn't exist in the peer members lists in other tc(tc1, tc2) anymore.